testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -634,6 +634,16 @@ def test_boolean_indexing(self):
         for at in [ag[0], ag[2]]:
             assert_equal(at in ag2, True)
 
+    def test_boolean_indexing_2(self):
```

Comment:
Other tests that are related to an issue are decorated with `@attr('issue')`. I do not know what the point is (especially since the ID of the issue is not reported), but since it seems to be a consistent trend throughout the test suite would it be worth decorating this test too?